code/game/objects/effects/proximity.dm

```diff
@@ -118,7 +118,8 @@
 	return
 
 /obj/effect/abstract/proximity_checker/Destroy()
-	monitor.checkers -= src
+	if (monitor.checkers)
+		monitor.checkers -= src
```

Comment:
if this is a lazylist you should use the macros, even if you hate them they at least prevent people from being dumb downline right?

Reply:
sure theyre just ugly

Reply:
watch this

```suggestion
	LAZYREMOVE(monitor.checkers, src)
```

Reply:
based?